packages/ra-language-french/index.js

```diff
@@ -40,6 +40,8 @@ module.exports = {
             loading: 'Chargement',
             not_found: 'Page manquante',
             show: '%{name} #%{id}',
+            empty: 'Pas encore de %{name}.',
+            invite: 'Voulez-vous en créer un ?',
```

Comment:
This will cause bad wording when the resource name is feminine (ex: promotion). But I don't see an easy way to fix that, so we'll go with that translation.

Reply:
We could leave to user to override the empty text, by looking for a `resources.foobar.empty` key. If there's no such key we use the default one `ra.page.empty`. 

The invite key could also be customizable, like `resources.foobar.invite`.

Reply:
Great idea. Would you mind adding that feature to your PR?

Reply:
I'll work on that